src/renderer/views/Trending/Trending.js

```diff
@@ -77,27 +78,35 @@ export default Vue.extend({
       }
     },
 
-    getTrendingInfoLocal: function () {
+    getTrendingInfoLocal: async function () {
```

Comment:
Should most of this method be moved to the api file as well? I imagine a function in the local api file could return the trending instance and the shown results. This could also help with decoupling local and invidious api from our components which would be useful if in the future we ever want to add support for other services (ex. PeerTube or Piped) or if we decide to switch to a different library in the future.

Let me know your thoughts on this 🙂 It can also be done in a future PR if you agree with this suggestion.

Reply:
Done the changes now, most of the logic is now in the api/local.js file. Please let me know if I didn't do it the way you wanted it.

Reply:
Yeah, the code lgtm. 🙂 Haven't tested it yet though.